src/OrchardCore.Modules/OrchardCore.Markdown/Drivers/MarkdownBodyPartDisplayDriver.cs

```diff
@@ -47,7 +47,6 @@ public override IDisplayResult Display(MarkdownBodyPart markdownBodyPart, BuildP
             return Initialize<MarkdownBodyPartViewModel>(GetDisplayShapeType(context), m => BuildViewModel(m, markdownBodyPart, context))
                 .Location("Detail", "Content:10")
                 .Location("Summary", "Content:10")
-                .Location("DetailAdmin", "Content:10") // For dashboard widgets
```

Comment:
as above

Reply:
Reusing existing Detail DisplayType and it's placement, so no need of defining DetailAdmin Display type and it's placement.

Reply:
I think the intention here was actually to have a different display type, so it could be customised for the admin if required.

Reply:
Revert back to use `DetailAdmin`